cli/pcluster/easyconfig.py

```diff
@@ -383,7 +373,86 @@ def scheduler_handler(scheduler):
         scheduler_info["max_size"] = "max_vcpus"
         scheduler_info["initial_size"] = "desired_vcpus"
     else:
-        scheduler_info["parameters_to_remove"] = "max_vcpus", "desired_vcpus", "min_vcpus", "compute_instance_type"
+        scheduler_info["parameters_to_remove"] = ("max_vcpus", "desired_vcpus", "min_vcpus", "compute_instance_type")
         scheduler_info["max_size"] = "max_queue_size"
         scheduler_info["initial_size"] = "initial_queue_size"
     return scheduler_info
+
+
+def _prompt_a_list(message, options, default_value=None):
+    """
+    Wrap prompt to use it for list.
+
+    :param message: the message to show the user
+    :param options: the list of item to show the user
+    :param default_value: the default value
+    :return: the validate value
+    """
+    if not options:
+        print("ERROR: No options found for {0}".format(message))
```

Comment:
what if we don't find options because the user is not authorized to list such resource but he is allowed to use it? We should allow the user to input the text for that option.